routes/volunteer_hours/volunteer_hours.js

```diff
@@ -0,0 +1,55 @@
+// endpoints related to volunteer hours
+const express = require('express');
+const pool = require('../../db');
+
+const hoursRouter = express();
+
+hoursRouter.use(express.json());
+
+// createSubmittedHours
+hoursRouter.post('/create', async (req, res) => {
+  try {
+    const { user_id, event_id, start_datetime, end_datetime, approved, notes } = req.body;
```

Comment:
🚫 **[eslint]** <[camelcase](https://eslint.org/docs/rules/camelcase)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Identifier 'end_datetime' is not in camel case.